website/templates/util/render_form.mako

```diff
@@ -2,14 +2,14 @@
     % if id:
         id="${id}"
     % endif
-    name="${name}" method="${method_string}" ${"action=\""+action_string+"\"" if action_string else ""} class="${form_class}">
+    name="${name}" method="${method_string}" action="${action_string}" class="${form_class}">
```

Comment:
Much better. I'm afraid to check to see if I did the terrible version you've replaced.